### ISSUE TITLE
Warnings in Guardian module

### ISSUE THREAD

**Issue by lbighetti** [lbighetti/hades#38]
I'm getting this when compiling the project

```elixir
warning: this clause cannot match because a previous clause at line 6 always matches
  lib/hades/guardian.ex:15

warning: this clause cannot match because a previous clause at line 19 always matches
  lib/hades/guardian.ex:27
```

**Comment by ccarvalho-eng** [comment]
That might the functions pattern matching. I got that module from the Guardian README but we can see a slightly different approach here: https://medium.com/@zacharykuhn/setting-up-a-phoenix-app-with-guardian-1-0-411ff3195adb. I'll take a look when I get a chance. Anyway that's not something we should care about but definitely should be investigated.

**Comment by lbighetti** [comment]
You fixed it on #44, closing this!
